datasets/index.md

```diff
@@ -20,23 +20,4 @@ Data in the Data Commons Graph comes from a variety of sources, each of which of
 9. [Health](/datasets/Health.html)
 10. [Housing](/datasets/Housing.html)
 
- <p> &nbsp; </p>
- 
-# Data coverage
-
-The following charts illustrate the data coverage in terms of global statistical variables at the country, state, and district levels. The first chart illustrates the total number of statistical variables available per country, excluding the USA where data coverage is currently most extensive
-
-<p> &nbsp; </p>
-<div class="flourish-embed flourish-map" data-src="visualisation/17709428"></div>
-<p> &nbsp; </p>
-
-This chart goes a level deeper and illustrates the total number of statistical variables available at the state level for each country worldwide.
-<p> &nbsp; </p>
-<div class="flourish-embed flourish-map" data-src="visualisation/17716780"></div>
-<p> &nbsp; </p>
-
-Finally, this third chart illustrates the total number of statistical variables available at the district/county level worldwide.
-
-<p> &nbsp; </p> 
-<div class="flourish-embed flourish-map" data-src="visualisation/17719383"></div>
-<p> &nbsp; </p>
```

Comment:
Is this deletion intentional? I think having these visualizations may have been an ask from James. @miss-o-soup

Reply:
good catch, I just ran the script & I don't think this should be deleted. Let me update

Reply:
done